Merge upstream missing commits

aliculPix4D opened the pull request:
- get the branch up to date with the upstream master

- [ ] I just noticed that it changed the hash of the two commits I made compared to the `pix4d-docker-updater` - should probably reject this PR, but want to check first - I used `git pull --rebase upstream master` but I don't think I should rebase

marco-m-pix4d commented:
This should be a real merge (we are managing a fork), if you pass `--rebase` to pull, it will not do what you want ...
I was expecting the source branch to be upstream/master, what is this test-merge-upstream instead ?

aliculPix4D commented:
> This should be a real merge (we are managing a fork), if you pass `--rebase` to pull, it will not do what you want ...
> I was expecting the source branch to be upstream/master, what is this test-merge-upstream instead ?
To explain the branch name. I have forked the `dependabot-core` repo from upstream, created a branch called `pix4d-docker-updater` from the correct upstream commit in July, after which I have added the changes from the current tip in `terraform-tomato/github-automation-tools` and verified that imported tests pass and that the behavior is the same as it is now. After this steps were done, I have made this branch a default one in the forked repo and added the branch protection rules. after that point I created an additional branch `test-merge-upstream` where I pulled the upstream changes with `git pull --rebase upstream master` and opened this PR. If it is alright with you I will reject this PR, create a new branch, pull the upstream changes but now with `git pull upstream master` and open a new PR?
@marco-m-pix4d

aliculPix4D commented:
Closing this PR. Taking different approach.
